James_Bond/james_bond.py

```diff
@@ -0,0 +1,40 @@
+
+
+
+name = input("What is your name?")
+user_input= input("Enter a list of random numbers separated by a comma:")
+user_list = user_input.split(",")
+
+for idx, num in enumerate(user_list):
+    print(len(user_list), num)
+
+    if len(user_list) >= 3:
+
+        if idx == len(user_list)-1:
+            print(f'{name} is not qualified to be James Bond!')
+
+        if num == '0'and user_list[idx+ 1] == '0' and user_list[idx + 2] == '7':
+            # print(num)
+            # print(user_list[idx+1])
+            # print(user_list[idx+2])
+            print(f'{name} is qualified to be James Bond!')
+            break
+        
+        else:
+            continue 
+    
+    else:
+        print(f'{name} is not qualified to be James Bond!')
+        break
+    
+                
```

Comment:
Unnecessary blank lines below. Only 1 blank line is acceptable at the end of every script.